reading of a file using fgets() .c

```diff
@@ -0,0 +1,17 @@
+#include<stdio.h>
+main()
+{
+  char str[10];
+  file *fp;
+  fp = fopen("f1.txt"."r");
+  if(fp == null)
+  { 
+    printf("file not found");
+    exist(1);
+  }
+  while(fgets(str,9,fp)!=null)       //fgets(str,n,fp)
+  {
+    printf("%s",str);
```

Comment:
If you are using the "if else" construct then subsequently the content of this block shall also be changed to:
 fgets(str, 9, fp); // read from file
 puts(str); // print read content out to stdout